client/verta/verta/_deployment/endpoint.py

```diff
@@ -133,6 +147,29 @@ def _get_json_by_path(cls, conn, workspace, path):
         return None
 
     def update(self, model_reference, strategy, wait=False, resources=None, autoscaling=None, env_vars=None):
+        """
+        Updates the Endpoint with a model logged in an Experiment Run or a Model Version.
+
+        Parameters
+        ----------
+        model_reference : `verta._tracking.experimentrun.ExperimentRun` or `verta._registry.modelversion.RegisteredModelVersion`
```

Comment:
When mentioning a class in docs, also add these (can be used just about anywhere):

- putting `:class:` right before the first backtick tells the docs generator to insert a link to that class's location in the docs, if available
- putting `~` right after the first backtick makes it so the rendered text is just `ExperimentRun` instead of `verta._tracking.experimentrun.ExperimentRun`, which can help with readability

```suggestion
        model_reference : :class:`~verta._tracking.experimentrun.ExperimentRun` or `verta._registry.modelversion.RegisteredModelVersion`
```